js/webrtc.js

```diff
@@ -931,6 +931,13 @@ var spreedPeerConnectionTable = [];
 			if (!OCA.SpreedMe.app.videoDisabled) {
 				OCA.SpreedMe.app.enableVideo();
 			}
+
+			var $hideVideoButton = $('#hideVideo');
+			if(OCA.SpreedMe.webrtc.webrtc.localStream.getVideoTracks() < 1) {
```

Comment:
Shouldn't that be `OCA.SpreedMe.webrtc.webrtc.localStream.getVideoTracks().length < 1`?